### DISCUSSION TITLE
Measure the forces on the deck，rather forces of the boat when floating

### DISCUSSION THREAD

**Discussion by hongshanma** [DualSPHysics/DualSPHysics#263] [DUALSPHYSICS_V5.2]
Help！！！
I want to measure the forces on the deck when floating， rather forces of the boat when floating. Deck is the part of the boat.
measure tool? gauge?or....

**Comment by alexbexe** [comment]
Using ComputeForces you can also select particles by id, so that you can select the id of those particles. That can be an option, but it depends how you are creating the deck of the boat.

Regards

**Reply by hongshanma** [reply]
Thank you for your reply.
I use drawfilestl to load the boat,. The id of deck's particles is difficult to know.
